tool/microkit/src/main.rs

```diff
@@ -565,8 +635,7 @@ fn get_arch_n_paging(region: MemoryRegion) -> u64 {
     // const PGD_INDEX_OFFSET: u64 = PUD_INDEX_OFFSET + 9;
 
     // get_n_paging(region, PGD_INDEX_OFFSET) +
```

Comment:
Oops, this should be removed instead of commented out. I'll add a PR to do that.